features/support/publishing_api_helpers.rb

```diff
@@ -1,14 +1,20 @@
 require "gds_api/test_helpers/publishing_api"
+require "gds_api/test_helpers/publishing_api_v2"
 
 module PublishingAPIHelpers
   include GdsApi::TestHelpers::PublishingApi
```

Comment:
I think we don't need `GdsApi::TestHelpers::PublishingApi` anymore. If we remove it we can get rid of the code in gds-api-adapters.